src/dedupe.coffee

```diff
@@ -71,6 +72,7 @@ class Dedupe extends Command
 
   forkDedupeCommand: (options, callback) ->
     dedupeArgs = ['--globalconfig', config.getGlobalConfigPath(), '--userconfig', config.getUserConfigPath(), 'dedupe']
+    dedupeArgs.push("--runtime=electron")
```

Comment:
That's a great point, @hedefalk. I considered using the `.apmrc` file to store all these configuration settings, but it felt somewhat inconsistent because we still need to use dynamically generated options (e.g. the architecture and the electron version) and there is something nice about having those options close to one another.

I guess an alternative could be to extract a method (and put it in `config.coffee`) named `getNpmBuildOptions()`/`getNpmBuildFlags()` which takes care of constructing an array with the necessary parameters. I'll open a pull-request with this change. 👍